chain/actors/policy/policy.go.template

```diff
@@ -132,6 +132,17 @@ func GetMaxProveCommitDuration(ver actors.Version, t abi.RegisteredSealProof) ab
 	}
 }
 
+func SetProviderCollateralSupplyTarget(num, denom big.Int) {
```

Comment:
Can we throw 

```
Can we throw 

```
// SetProviderCollateralSupplyTarget sets the percentage of normalized circulating
// supply that must be covered by provider collateral in a deal. This should
// only be used for testing.
```